.github/workflows/integration.yml

```diff
@@ -0,0 +1,64 @@
+name: integration-test
+
+on:
+  schedule:
+    - cron: '0 0 * * 6'
+  workflow_dispatch:
+  
+  
+jobs:
+  integration:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip:ci') }}
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 2
+        submodules: true
+    - name: Apply cache for Pants
+      uses: actions/cache@v3
+      id: cache
+      with:
+        # pants-specific cache
+        path: |
+          ~/.cache/pants/setup
+          ~/.cache/pants/lmdb_store
+          ~/.cache/pants/named_caches
+        key: ${{ runner.os }}-${{ hashFiles('pants*.toml', '**/*.lock') }}-
```

Comment:
Every test reuses this cache?
And how long this cache remain?

Reply:
https://github.com/lablup/backend.ai/actions/caches
Recently updated github to check caches in .

I don't know exactly how much cache is maintained, but I checked until the cache was over a month old.

Reply:
https://github.com/actions/cache
Caches that have not been accessed within the past week are deleted, and if the 10GB capacity is full, the oldest cache is deleted.